README.md

```diff
@@ -1 +1,217 @@
-# Hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <h3><b>Hello Microverse</b></h3>
+</div>>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+- 
+<!-- PROJECT DESCRIPTION -->
+# 📖 [Hello-world] <a name="about-project"></a>
+This is a simple project done for Microverse. Through this project I have learned how to prepare a GitHub Repository for a project in a more Professional way. 
+**[Hello Microverse]** 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+Not applicable for the project
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Not applicable</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Not applicable</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Not applicable</a></li>
+  </ul>
+</details>
```

Comment:
- [ ] [OPTIONAL] All software applications whether big or small have a tech stack 
>  A technology stack, also known as a software stack or development stack, is a data ecosystem that enlists the underlying tools, frameworks, and libraries used to build and run your application. 

The Tech Stack of `Hello World` can include HTML/CSS, Linters, Git, Github etc

Reply:
I have removed the links under the Tech Stack. However, it will very kind of you if you let me know how can I test my own project to check if the linter is working or not.

Reply:
@shella12 I tried to check everything if the file structure is wrong. I have not found anything wrong. I tried to run the whole procedure of linter, lighthouse, webhint, and Styleint in my local console and I did not find anything wrong. However, I have already corrected the Readme file as you mentioned. I am lost dont know where to check now? Should I delete the whole project and redo it again???